storagegroup/storagegroup.go

```diff
@@ -206,3 +211,84 @@ func formatCheck(v2 *storagegroup.StorageGroup) error {
 
 	return nil
 }
+
+// ReadFromObject assemble StorageGroup from a regular
+// Object structure. Object must contain unambiguous information
+// about its expiration epoch, otherwise behaviour is undefined.
+//
+// Returns any error appeared during storage group parsing.
+func ReadFromObject(sg *StorageGroup, o objectSDK.Object) error {
```

Comment:
Why is this not a method? Would eliminate possible confusion about the argument order.

Reply:
because these funcs do not use any private methods/fields of the `StorageGroup` struct. that rule was suggested by the @cthulhu-rider